### DISCUSSION TITLE
Is the variable font still supported?

### DISCUSSION THREAD

**Discussion by FloEdelmann** [IBM/plex#470] [Q&A]
To save bandwidth in our web app, we use IBM Plex Sans Var (or Variable) from https://github.com/IBM/plex/tree/master/IBM-Plex-Sans-Variable instead of including multiple non-variable weights/styles.

While the non-variable sans-serif font was [released 20 days ago](https://github.com/IBM/plex/releases/tag/v6.1.0) and is now at version 3.4, the variable font [didn't receive any commits](https://github.com/IBM/plex/commits/master/IBM-Plex-Sans-Variable) since its initial v1.0 commit in April 2019. Thus, the question: Is the variable font (still) supported?

In the [to-dos for the upcoming v7.0](https://github.com/IBM/plex/issues/370), the variable font is explicitly mentioned:

> Create declarations for Variable font

It would be awesome if the various improvements to IBM Plex Sans could also be added to the variable font as well!

**Comment by tay1orjones** [comment]
cc @BoldMonday

**Reply by BoldMonday** [reply]
An update to Plex Sans Var is on our to-do list. A number of bugs have been reported and we intend to merge all additions and improvements from the static Plex Sans fonts. We expect to deliver this in Q4 2022.
